activesupport/lib/active_support/testing/assertions.rb

```diff
@@ -32,6 +32,7 @@ def assert_not(object, message = nil)
       #   end
       def assert_nothing_raised
         yield
+        assert true
```

Comment:
This is changing the return of this method to `true`. We need to probably hold the value of yield and return after the assert.

Reply:
Oh, right, we do need to keep the return value of the block for sure
Let me do the change

Reply:
Changed!